Packs/AWS-EC2/Integrations/AWS-EC2/AWS-EC2.py

```diff
@@ -170,26 +170,28 @@ def run_command(account_id: str) -> CommandResults:
 
 
 def test_module() -> str:
+    client = build_client({})
+    response = client.describe_regions()
+    if response['ResponseMetadata']['HTTPStatusCode'] != 200:
+        raise DemistoException(f'Test Module failed. Response: {response}')
     if ROLE_NAME:
         if not PARAMS.get('accounts_to_access'):
-            raise DemistoException("'AWS organization accounts to access' must not be empty when an access role is provided.")
+            raise DemistoException("'AWS organization accounts' must not be empty when an access role is provided.")
 
         def test_account(args: dict) -> CommandResults:
             build_client(args)
-            return CommandResults(readable_output='ok')
+            return CommandResults()
         fails = [
-            result.readable_output.split('`')[1]
+            result.readable_output
             for result in run_on_all_accounts(test_account)({})  # type: ignore
             if result.entry_type == EntryType.ERROR
         ]
         if fails:
+            demisto.debug('\n\n'.join(fails))
+            fail_ids = ', '.join(res.split('`')[1] for res in fails)
```

Comment:
add a comment here what result.readable_output format is and why this makes sense